packages/mobile/README.md

```diff
@@ -88,6 +88,18 @@ that you want to use with the app, you can run
 yarn run build-sdk TESTNET
 ```
 
+## Migrating networks
+
+To migrate to a new network, update the environment configuration files within the app and its dependencies:
+
+Update `.env.ENV-NAME` and `packages/mobile/.env.ENV-NAME` with the new network name and settings. 
+
+For the blockchain api, update `packages/blockchain-api/app.ENV-NAME.yaml` with the new contract addresses, which can be found using `celotooljs` or by looking at the address of the smart contracts in `/packages/protocol/build/ENV-NAME/contracts`.
```

Comment:
Appreciate the update here but not sure this is the best place to have details about blockchain api and notification service.
1. Don't think these details are relevant to external devs
2. This should probably live in a bigger "How to bring up a testnet" doc :)

Reply:
Great point! Can you link the how to bring up a testnet doc?

Reply:
It needs work but here's what we've got so far https://docs.google.com/document/d/1vMXZY6pu2fmwHxTzZE0YbKhn6Gs9szPZjHtjvV8dsis/edit#heading=h.ybn9lro53cxv